README.md

```diff
@@ -111,6 +110,14 @@ This will:
 * increment the version, commit the change, and create a git tag
 * push the changes and the new tag to the remote repo
 
+### Deploy to AWS
+
+From main, `npm run build:prod` to create the `/dist` folder. Upload the contents of `/dist` to AWS. (The plan is to get a Jenkins job to automate this part.)
```

Comment:
```suggestion
From main, `npm run build:prod` to create the `dist/` folder. Upload the contents of `dist/` to AWS. (The plan is to get a Jenkins job to automate this part.)
```
`/folder` generally means from the root directory, so I'd leave off the prepended slash from dir names when possible unless it is an absolute path.